package.json

```
@@ -36,7 +36,7 @@
     "commit": "cz",
     "doctoc": "doctoc README.md packages/collector/README.md",
     "lint": "node misc/is_eol || (eslint .*.js *.js && npm run typecheck && lerna run lint --stream && eslint misc && eslint dockerfile-examples)",
-    "postinstall": "husky install && npm run bootstrap",
+    "postinstall": "./bin/install-optional-dev-dependencies.sh && husky install && npm run bootstrap",
```

_nitpick:_ this could be
```suggestion
    "postinstall": "bin/install-optional-dev-dependencies.sh && husky install && npm run bootstrap",
```